platform/windows/display_server_windows.cpp

```diff
@@ -4311,7 +4339,7 @@ LRESULT DisplayServerWindows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARA
 				break;
 			}
 
-			DisplayServer::WindowID receiving_window_id = _get_focused_window_or_popup();
+			DisplayServer::WindowID receiving_window_id = get_window_at_screen_position(mouse_get_position());
```

Comment:
This is what breaks the drag and drop, reverting it back to `_get_focused_window_or_popup()` seems to make mouse motion works for drag and drop, but not the mouse release. also the CanvasItemEditor doesn't recieve the input while drawing a selection rect and entering another godot window.